docs/recipes/build/docker-image.md

```diff
@@ -128,10 +125,10 @@ RUN npm install
 COPY webpack.config.js ./
 COPY src/Shared src/Shared
 COPY src/Client src/Client
-RUN npm run build
+RUN dotnet fable src/Client --run webpack
 
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim
```

Comment:
Any reason to change to `buster-slim` instead of `alpine`? Not a big deal either way as long as it's consistent with [line 42](https://github.com/SAFE-Stack/docs/pull/215/files#diff-88f59a8c96134a4be899af3fccc48ce840abcaaf85db2218eed65cb0a99c3ce6R42)

Reply:
No reason I was just getting the image working, just tested out alpine now and all good :)